src/vs/workbench/api/node/extHostDebugService.ts

```diff
@@ -3,8 +3,6 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// {{SQL CARBON EDIT}}
```

Comment:
I had to remove these edits since the formatting does weird things to this file. I added edits in other places to account for these removes.